### DISCUSSION TITLE
Permissions checkboxes not rendering well

### DISCUSSION THREAD

**Discussion by gfo2007** [serenity-is/Serenity#6539] [Help] [answered]
Hi,
Below is the image of the Permissions dialog box. The checkboxes that allows a user to select permissions are not rendering well in the UI  for them to be ticked by the user.

Am I missing a library or is a library not loading well? 

![permissions rendering](https://user-images.githubusercontent.com/14317420/191059976-be777b80-6f31-4ab2-b76f-5502a75af22b.PNG)

**Comment by VictorTomaili** [comment, marked as answer]
I created an issue about this. #6542
